.github/tests/test_script.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+
+# Copyright (C) 2020 Vanessa Sochat.
+
+# This Source Code Form is subject to the terms of the
+# Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+# with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+import json
+import os
+import re
+import pytest
+import yaml
+
+here = os.path.dirname(os.path.abspath(__file__))
+root = os.path.dirname(os.path.dirname(here))
+
+
+def load_schema(path):
```

Comment:
I can see this method `load_schema`  be generic to all schema such that we can have some method that can be imported. That we all regtest can use the same method instead of redefining it for every file.

It could be something like ``from utils.schema import load_schema``.

Reply:
Yep, this will definitely be the case for buildtest - for the tests here we just include in the file because there isn't a module to import from. It's a fairly simply function so I think this strategy is okay (it's unlikely to change or get complicated).